api/transport/errors.go

```diff
@@ -49,3 +51,17 @@ func IsTimeoutError(err error) bool {
 	_, ok := err.(errors.TimeoutError)
 	return ok
 }
+
+// UnrecognizedProcedureError returns an error for the given request,
+// such that IsUnrecognizedProcedureError can distinguish it from other errors
+// coming out of router.Choose.
+func UnrecognizedProcedureError(req *Request) error {
```

Comment:
Can we stop adding errors to `api/transport/errors.go` per the work I'm doing in #1064?